recipes/matam/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "matam" %}
+{% set version = "1.6.0" %}
+{% set sha256 = "unused" %}
+
+package:
+  name: matam
+  version: {{ version }}
+
+source:
+  # can't use url due to submodules not present in github release
+  # url: https://github.com/bonsai-team/{{ name }}/archive/v{{ version }}.tar.gz
+  git_url: https://github.com/bonsai-team/matam.git
+  git_tag: v{{ version }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - cmake
+    - autoconf
+    - automake
+
+  host:
+    - zlib
+    - python >=3
```

Comment:
Is python actually needed for installation? I see there is one script, but that shouldn't need to be compiled.

Reply:
Yes, python is not needed for the installation.